### PR TITLE
chore: fix check for prettier formatted files in CI

### PR DESCRIPTION
**Pull request by fiam**
Tell prettier to ignore unknown extensions, and also exclude .html
files since we do not run those by prettier

### CODE REVIEW

File: .github/workflows/packages-ci.yaml [hunk 1]
```diff
@@ -93,7 +93,9 @@ jobs:
           main-branch-name: ${{ steps.branch-name.outputs.current_branch }} # Get the last successful commit for the current branch.
 
       - name: Check for formatting errors
-        run: eval "pnpx prettier --check $(git diff --name-only --diff-filter=ACMRT -r ${{ steps.last_successful_commit_push.outputs.base }} | xargs)"
+        # Ignore unknown file extensions for prettier as well as files with .html extension, since
+        # we don't format those with prettier.
+        run: eval "pnpx prettier --check -u $(git diff --name-only --diff-filter=ACMRT -r ${{ steps.last_successful_commit_push.outputs.base }} | grep -v '\.html$' | xargs)"
```
Comment (StarpTech) [inline .github/workflows/packages-ci.yaml:98]: IMO this is the wrong approach. Prettier has an ignore file. We need to exclude patterns here https://github.com/wundergraph/wundergraph/blob/main/.prettierignore

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: .github/workflows/packages-ci.yaml (hunk 1 above)
Comment (StarpTech) [reply .github/workflows/packages-ci.yaml:98]: Prettier has also an https://prettier.io/docs/en/cli.html#--ignore-unknown option

Comment (fiam) [reply .github/workflows/packages-ci.yaml:98]: It seems to ignore `.prettierignore` for command line arguments. Let me give it another try. `--ignore-unknown` is in the PR, `-u` is its short form.

Comment (StarpTech) [reply .github/workflows/packages-ci.yaml:98]: Or try this https://github.com/azz/pretty-quick, which should work.

Comment (fiam) [reply .github/workflows/packages-ci.yaml]: @StarpTech I'd rather not take a dependency when we're 95% there. It should be good now, PTAL.

Comment (StarpTech) [reply .github/workflows/packages-ci.yaml]: If it works fine! :smile:

**Review by StarpTech — APPROVED** [review]
LGTM

File: .github/workflows/packages-ci.yaml (hunk 1 above)
Comment (fiam) [reply .github/workflows/packages-ci.yaml]: Thanks 👍

**Review by jensneuse — APPROVED** [review]

**Comment by jensneuse** [comment]
Thank you @fiam !

**Pull request merged**